sentry/src/main/java/io/sentry/config/PropertiesProvider.java

```diff
@@ -58,8 +58,20 @@ default String getProperty(@NotNull String property, @NotNull String defaultValu
    * @return property value or the default value if not found.
    */
   @Nullable
-  default Boolean getBooleanProperty(@NotNull String property) {
+  default Boolean getBooleanProperty(final @NotNull String property) {
     final String result = getProperty(property);
     return result != null ? Boolean.valueOf(result) : null;
   }
+
+  /**
+   * Resolves a {@link Double} property given by it's name.
+   *
+   * @param property - the property name
+   * @return property value or the default value if not found.
+   */
+  @Nullable
+  default Double getDoubleProperty(final @NotNull String property) {
+    final String result = getProperty(property);
+    return result != null ? Double.valueOf(result) : null;
```

Comment:
should we account if it's not a Double? like catch `NumberFormatException` and return null?

Reply:
Given this is done at startup and the value must be double, probably OK to blow up here

Reply:
that's o but kinda goes against our philosophy

https://develop.sentry.dev/sdk/philosophy/#degrade-gracefully

I'd prefer blowing up only if there's no DSN, any other reason would be a bug IMO